src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -77,6 +91,11 @@ import getPartitionCookies from 'backend/utils/get_partition_cookies'
 import { prepareBaseGameForModding } from 'backend/ipcHandlers/mods'
 import { runWineCommandOnGame } from 'backend/utils/compatibility_layers'
 
+import { chmod, writeFile } from 'fs/promises'
+import { trackEvent } from 'backend/api/metrics'
```

Comment:
```suggestion
import { trackEvent } from 'backend/metrics/metrics'
```

without this, we are attempting to send the event via the ipcRenderer and get an error message like this in the console

```
TypeError: Cannot read properties of undefined (reading 'invoke')
    at trackEvent 
    at applyPatching 
    at async Module.update$2 [as update] 
    at async updateQueueElement 
    at async initQueue 
```

Reply:
true, autoimports sometimes imports from the api calls. will fix